### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by AndreMiras** [AndreMiras/PyWallet#19]
`pyethapp` currently requires `gipc==0.4.0` which is not Python3 compatible.
Python 3 support was added to `gipc==0.6.0`.
See pip install error output below:
```
Collecting gipc==0.4.0 (from pyethapp==1.5.0->-r requirements.txt (line 7))
  Downloading gipc-0.4.0.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-ivs7zndf/gipc/setup.py", line 22, in <module>
        long_description=open("README.rst").read().decode('utf-8'),
    AttributeError: 'str' object has no attribute 'decode'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ivs7zndf/gipc/
```

**Comment by AndreMiras** [comment]
Upstream issue: https://github.com/ethereum/pyethapp/issues/214

**Comment by AndreMiras** [comment]
Fixed in  989784b
